bin/pycbc_inspiral

```diff
@@ -187,7 +189,10 @@ fft.verify_fft_options(opt,parser)
 pycbc.opt.verify_optimization_options(opt, parser)
 pycbc.weave.verify_weave_options(opt, parser)
 
-pycbc.init_logging(opt.verbose)
+if opt.info:
```

Comment:
My intention was to intervene as little as possible with existing behavior. E.g. 'init_logging' now, apart from 'True' and 'False' as before, also takes a numerical log level as first argument. I could modify pycbc_inspiral to count the '--verbose' options and pass a numeric log level based on that number. However this would change the behavior of pycbc_inspiral if a single --verbose is given. To get the full verbose output you would now need to specify --verbose twice. Would you prefer that?